examples/step-22/doc/intro.dox

```diff
@@ -19,17 +19,17 @@ California Institute of Technology.
 This program deals with the Stokes system of equations which reads as
 follows in non-dimensionalized form:
 @f{eqnarray*}
-  -\textrm{div}\; \varepsilon(\textbf{u}) + \nabla p &=& \textbf{f},
+  -2\; \textrm{div}\; \varepsilon(\textbf{u}) + \nabla p &=& \textbf{f},
   \\
   -\textrm{div}\; \textbf{u} &=& 0,
 @f}
 where $\textbf u$ denotes the velocity of a fluid, $p$ is its
 pressure, $\textbf f$ are external forces, and
-$\varepsilon(\textbf{u})= \nabla^s{\textbf{u}}= \frac 12 \left[
+$\varepsilon(\textbf{u})= \nabla^s{\textbf{u}}= \left[
```

Comment:
The definition of the strain rate tensor varepsilon(u) should still contain the factor 1/2 (otherwise you change the equation by a factor of 4).